about.html

```diff
@@ -1,73 +1,92 @@
 <!doctype html>
 
 <html lang="en">
-<head>
-  <meta charset="utf-8">
+    <head>
+        <meta charset="utf-8">
 
-  <title>Sprint Challenge - About</title>
+        <title>Sprint Challenge - About</title>
 
-  <link href="https://fonts.googleapis.com/css?family=Roboto|Rubik" rel="stylesheet">
-  <link rel="stylesheet" href="css/index.css">
+        <link href="https://fonts.googleapis.com/css?family=Roboto|Rubik" rel="stylesheet">
+        <link rel="stylesheet" href="css/index.css">
 
-</head>
+    </head>
 
-<body>
-    <div class="container about-page">
-     
-    About
 
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
+    <body>
+        <header>
 
 
-    <img src="img/about-plan.png" alt="strategy">
-    
-    Strategy
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
+            <div class="header">
+                <nav>
+                    <img src="img/lambda-black.png">
+                    <a href="index.html">Home</a>
+                    <a href="About">About</a>
```

Comment:
In line 23, you need to link the about.html in order to get the about-us page to display when on click.